.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Describe a bug you've seen or make a case for a new feature
```

Comment:
"... or make a case for a new feature" should probably be deleted as there is a separate feature request form?